core/src/main/java/org/apache/carbondata/core/util/CarbonUtil.java

```diff
@@ -3177,4 +3177,34 @@ public static void setLocalDictColumnsToWrapperSchema(List<ColumnSchema> columns
     }
     return columnLocalDictGenMap;
   }
+
+  public static DataType getMappingDataType(String type) {
```

Comment:
Please reuse org.apache.carbondata.core.util.DataTypeUtil#valueOf(java.lang.String)  this method.. This seems to be duplicate method

Reply:
done